manifests/d/DuckStudio/GitHubLabelsManager/1.5/DuckStudio.GitHubLabelsManager.locale.en-US.yaml

```
@@ -0,0 +1,49 @@
+# Created with komac v2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.6.0.schema.json
+
+PackageIdentifier: DuckStudio.GitHubLabelsManager
+PackageVersion: '1.5'
+PackageLocale: en-US
+Publisher: DuckStudio
+PublisherUrl: https://duckduckstudio.github.io/yazicbs.github.io/
+PublisherSupportUrl: https://github.com/DuckDuckStudio/GitHub-Labels-Manager/issues
+# PrivacyUrl:
+Author: 鸭鸭「カモ」
+PackageName: GitHub Labels Manager
+PackageUrl: https://github.com/DuckDuckStudio/GitHub-Labels-Manager
+License: GNU General Public License v3.0
+LicenseUrl: https://github.com/DuckDuckStudio/GitHub-Labels-Manager/blob/main/LICENSE
+Copyright: Copyright (c) DuckStudio 鸭鸭「カモ」
+CopyrightUrl: https://github.com/DuckDuckStudio/GitHub-Labels-Manager/blob/main/LICENSE
+ShortDescription: A GitHub Labels Manager
+Description: Tools that automatically help you copy repo labels, get repo labels, and empty existing labels.
+# Moniker: glm
+Tags:
+- devtools
+- github
+- github-labels
+- labels
+ReleaseNotes: |
+  *感谢所有为本版本做出贡献的人! | Thanks for all contributions*  
+  > @DuckDuckStudio
+
+  > [!NOTE]
+  > 本版本由[自动流](https://github.com/DuckDuckStudio/GitHub-Labels-Manager/actions/runs/12078168910)构建，经重命名后发布。
+
+  > [!TIP]
+  > 本版本为修复 (Fix)
+
+  ## 修复 (Fix)
+  - [x] fix: 修正 422 与 401 错误返回 (Fix error return of 422 and 401 status codes.)
+
+  ---
+
+  **Full Changelog**: https://github.com/DuckDuckStudio/GitHub-Labels-Manager/compare/v1.4...v1.5
+ReleaseNotesUrl: https://github.com/DuckDuckStudio/Chinese_git/releases/tag/v1.5
```

```suggestion
ReleaseNotesUrl: https://github.com/DuckDuckStudio/GitHub-Labels-Manager/releases/tag/v1.5
```

Is it another "accident"? 😄

---

Thank you for catching this issue!